src/claimApplication/header.js

```diff
@@ -8,11 +8,16 @@ export function replaceHeader(pageId) {
   if (screensWithoutTabs.includes(pageId)) {
     replacePopulatedHeader();
   } else {
+    let headerReady = false;
     const observer = new MutationObserver((mutations) => {
       mutations.forEach(() => {
+        if (headerReady) return;
         const tabsDiv = document.querySelector('.ajax__tab_header');
         if (tabsDiv) {
+          headerReady = true;
           replacePopulatedHeader();
+          const event = new CustomEvent('headerReady');
+          document.dispatchEvent(event);
           observer.disconnect();
```

Comment:
this allows `certification.js` to reset the title after the header has been created (trying to do so right away is too early).

also the boolean is needed because several mutations (it was like 21 when I tested) were observed by the observer before it disconnected and each one caused the header to re-render. Now it only gets rendered once!